src/support/timefuncs.c

```diff
@@ -28,31 +28,32 @@
 extern "C" {
 #endif
 
-#if defined(_OS_WINDOWS_)
-static double floattime(void)
+int jl_gettimeofday(struct jl_timeval *jtv)
```

Comment:
Yes, I think having it in the header is sufficient.  However, stylistically you'll notice that most of the `src/*.c` functions use `DLLEXPORT` in the `.c` files as well as in the header.   I think that is just to make it clearer when reading the source code that the function is intended to be exported.

Would be good to follow that style here.  And if `clock_now` is exported, it should probably be `jl_clock_now`.